manifests/c/ChristianSchenk/MiKTeX/22.3/ChristianSchenk.MiKTeX.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ChristianSchenk.MiKTeX
+PackageVersion: "22.3"
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: user
+InstallModes:
+- interactive
+- silentWithProgress
+InstallerSwitches:
+  Silent: --unattended
+  SilentWithProgress: --unattended
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://mirror.kumi.systems/ctan/systems/win32/miktex/setup/windows-x64/basic-miktex-22.3-x64.exe
```

```suggestion
  InstallerUrl: https://miktex.org/download/ctan/systems/win32/miktex/setup/windows-x64/basic-miktex-22.3-x64.exe
```
If that mirror is unavailable then the URL will not work.

---

Are redirects for CDNs not supported?